docs/contributing.rst

```diff
@@ -132,6 +132,29 @@ mean that this work should be relatively manageable.
 We hope that the lessons we learn building something useful in the short term will be
 applicable in the future as other needs arise.
 
+.. _backwards compatibility with meica:
+
+Is ``tedana`` backwards compatible with MEICA?
+``````````````````````````````````````````````
+
+The short answer is no.
+
+There are two main reasons why. One is that the tool originally used to run the independent
+component analysis core to the MEICA method (`mdp`_) is no longer supported. In November 2018
+the developers made the decision to switch to `scikit learn`_ to perform these analyses. Scikit
+learn is well supported and under long term development. ``tedana`` will be more stable and
+have better performance going forwards as a result of this switch, but it also means that
+exactly reproducing MEICA analyses is not possible.
+
+The other reason is a choice of the core developers to look forwards rather than maintaining
+an older codebase. As is described `above`_, ``tedana`` is developed by a small team of
+volunteers and they have to allocate their time accordingly. If you'd like to use MEICA as has
+been previously published the code is available on `github`_ and freely available under a LGPL2
```

Comment:
I think we should actually link to the bitbucket here, rather than the github, since the version on github has actually not previously been published on !

Reply:
So, the bitbucket link says to go to the github repo....and I figured if someone is trying to use MEICA it would be super annoying to have to go to bitbucket just to then be sent to the github link......

But I know what you mean, so happy to change if you'd like?

Reply:
This is the only part of your review I haven't done @emdupre - let me know what you think. This is one of my only moderately held opinions....

Reply:
I see your point here, for sure -- really this is a problem in the bitbucket. We can add it to our to-do list once https://github.com/ME-ICA/tedana/pull/119 is merged and the two do really differ

Reply:
Coolio! Updated  - I'll push that change now ✨